lib/taskana-core/src/test/java/acceptance/task/QueryTaskWithAttachmentAccTest.java

```diff
@@ -84,8 +84,7 @@ void should_ConfirmIfAttachmentSummariesAreCorrect_When_UsingTaskQueryAndGetTask
             .collect(Collectors.toList());
 
     assertThat(queryAttachmentSummaries)
-        .hasSize(originalAttachments.size())
-        .containsExactlyElementsOf(originalAttachments) // same values
+        .containsExactlyInAnyOrderElementsOf(originalAttachments) // same values
```

Comment:
@gitgoodjhe: for some reason the ordering changed (and is somehow different running the tests on travis compared to my local machine.. This is a really weird behaviour, but ignoring the order of elements works :+1: )